content/en/docs/Concepts/MVC.md

```diff
@@ -14,11 +14,11 @@ The Model-View-Controller (MVC) pattern consists of three separate branches. The
 
 MVC has many very well documented benefits. It's strongly defined structures allow for good organization when dealing with large projects. MVC's structure also allows for asynchronous method invocation, meaning methods can be called and the rest of the program is able to be run while waiting for the called method to finish. This allows for faster loading applications. These methods can be run by using the keyword "await" to call a method that has been defined with the "async" property.
 
-Because the MVC format is so separated, people can work on different sections of the code at the same time with very little complications occurring. This separation also makes debugging more simple.
+Because the MVC format is separated into multiple sections, people can work on different sections of the code simultaneously without causing issues with the code. This separation can also make debugging simpler.
 
 ## Routing
 
-Routing in MVC usually follows a specific convention, but it can be done nearly any way by altering startup code. When making a project with MVC in Microsoft Visual Studio 2022 using ASP.NET Core 6, the default routing is the following:
+Routing in a MVC usually follows a specific convention, but it can be done nearly any way by altering the startup code. When making a project with a MVC in Microsoft Visual Studio 2022 using ASP.NET Core 6, the default routing is the following:
```

Comment:
Remove the specification of VS 2022, the process is the same regardless of the IDE.